plugins/modules/aci_l3out_bgp_peer.py

```diff
@@ -146,6 +146,7 @@
   bgp_password:
     description:
     - Password for the BGP Peer.
+    - Providing the password will always result in a change because the set password cannot be retrieved from APIC.
```

Comment:
Should we also add it to the other password descriptions? I think there's about 6. I found them using this quick search - `password.*no_log=True`

Reply:
I do agree with @samiib, I think we should add it to every password descriptions.

Reply:
@samiib, I think the issue then gets bloated but we should create a new issue if we want to clean out the entire repository. The reason is that I think your quick search is too limited in my opinion and might also result in code changes and thus also test adjustments for all these found cases.

I think the search should be done on every module because we have more attributes marked with no_log and we can also not guarantee that all these attributes are marked correctly. Furthermore there are keys, pass-phrases, perhaps others arguments present in the repository which might present similar behaviour.

Reply:
@akinross True. It could be much larger than anticipated.
We can just add the description to new modules or if someone raises a similar issue.

Reply:
It's not just the passwords but as @akinross  mentioned even the keys are not returned by APIC so we would have to add descriptions there as well. I agree that a new issue is required for such a change.